public/js/constants/atomData.js

```diff
@@ -90,9 +90,15 @@ export const chart = {
   description: "A variety of different charts"
 };
 
+export const audio = {
+  type: "audio",
+  fullName: "Audio",
+  description: "A small audio player"
+};
+
 
-export const allAtomTypes = [cta, recipe, storyQuestions, quiz, media, qa, guide, profile, timeline, explainer, commonsDivision, chart];
-export const workshopEditableAtomTypes = [cta, storyQuestions, recipe, qa, guide, profile, timeline, explainer, commonsDivision, chart];
+export const allAtomTypes = [cta, recipe, qa, guide, profile, timeline, media, chart, audio,  storyQuestions, quiz, explainer, commonsDivision];
+export const workshopEditableAtomTypes = [cta, storyQuestions, recipe, qa, guide, profile, timeline, explainer, commonsDivision, chart, audio];
```

Comment:
I reordered these to put the most used ones first. This means they come at the top on the UI.

Reply:
👍